week-1-practise/level-2/4-currency-conversion-2.js

```diff
@@ -7,8 +7,13 @@
   Find a way to add 1% to all currency conversions (think about the DRY principle)
 */
 
-function convertToUSD(price) {}
-function convertToBRL(price) {}
+function convertToUSD(price) {
```

Comment:
we want to add a 1% fee for USD conversion as well. Can we share that logic using nested functions?